client/js/widgets/widget.js

```diff
@@ -602,27 +602,30 @@ export class Widget extends StateManaged {
         }
       }
       if(a.func == 'IF') {
-        if(typeof a.variable != 'undefined' && Array.isArray(a.thenRoutine)) {
+        setDefaults(a, { relation: '==' });
+        if (!['==', '!=', '<', '<=', '>=', '>'].indexOf(a.relation)) {
```

Comment:
Did you test this? I didn't but the condition should be `true` for `==` (index `0`) and `false` for everything else (index `-1` or positive).